### DISCUSSION TITLE
Noob Questions

### DISCUSSION THREAD

**Discussion by kevmeist3r** [nmelhado/league-page#157] [Q&A]
First off, just wanted to say how awesome this is so thank you for all the hard work!

I'm somewhat familiar with coding, but a total noob doing at using GitHub and making these changes on my own. After using the training wheels guide and poking around for a few hours I think I'm getting familiar with everything but I wanted to reach out to the community to see if I could be pointed in the right direction:

- Where should I look to be able to replace the "League Page 2021" icon on the top of the home page?

- I'd like to integrate my own power rankings, so I would like to know where to look to be able to modify/remove the existing Rest of Season power rankings chart on the home page

- Lastly, has anyone else had any issue with the Managers pages where it says "Retrieving players..." and never finishes loading the current roster or Team transactions? I deleted my fork and restarted because I thought I might have ruined something, but the problem persists with my new fork also. Wanted to check with the community before I submit a bug.

Thanks!

**Comment by mdandrews** [comment]
I’m having the same issue with the rosters. 

Also I have no idea how to enable comments on the blog posts.

this is such an amazing tool but I’m a complete coding noob!

**Comment by dawghouse73** [comment]
I know it's been a minute, but has anyone heard on this? I am new to this over the last few weeks and have come across a few things as well that I'm not certain. 

The League Page.png image - I uploaded my own file named League Page.png with our own league logo, but it won't seem to overwrite. 

And I'm having a similar, yet different issue with the Manager's page - mine just loads a blank page. And to make matters worse, if I access the Manager's page, all other links become dead and I'm forced to refresh the browser in order to access any other pages. 

Appreciate any insight you may have compiled yourself since you guys posted.

Cheers.

**Reply by nmelhado** [reply]
@dawghouse73 If you want to replace the logo, follow the instructions at the bottom of the training wheels guide.

> If you want to replace the league page logo with your league's own logo, replece static/badge.png with your own 120px x 120px png file (using the same name)

**Reply by dawghouse73** [reply]
Awesome. Thanks. I was trying to replace the favicon instead. I cannot sing the praises of this tool to my league mates enough. 👍🏻👍🏻👍🏻
